mlos_bench/mlos_bench/service/config_persistence.py

```diff
@@ -45,7 +48,10 @@ def __init__(self, config: Optional[Dict[str, Any]] = None,
         """
         super().__init__(config, parent)
         self._config_path = self.config.get("config_path", [])
-        self._config_loader_service = self
+        self.config_loader_service = self
+
+        if self.BUILTIN_CONFIG_PATH not in self._config_path:
+            self._config_path.append(self.BUILTIN_CONFIG_PATH)
```

Comment:
This is a key part of this: since we're shipping the configs as part of the module now, we should be able to locate the those stock configs/scripts via the module directly.